app/src/main/java/com/cornellappdev/android/eatery/Model/CafeteriaModel.java

```diff
@@ -126,11 +122,12 @@ public String isOpen(){
             HashMap<Integer, ArrayList<Date>> hours = cafeInfo.getHoursH();
             if(hours.containsKey(day)){
                 foundDay = true;
-                int startT = hours.get(day).get(0).getHours() + hours.get(day).get(0).getMinutes();
-                int endT = hours.get(day).get(1).getHours() + hours.get(day).get(1).getMinutes();
-                int curT = now.getHours() + now.getMinutes();
+                int startT = hours.get(day).get(0).getHours()*60 + hours.get(day).get(0).getMinutes();
+                int endT = hours.get(day).get(1).getHours()*60 + hours.get(day).get(1).getMinutes();
+                int curT = now.getHours()*60 + now.getMinutes();
+                System.out.println(now.getHours());
                 if(curT>=startT && curT<endT){
```

Comment:
I know that I can try to understand this line, but just add a comment so that I don't need to think